tests/vtpm/test_vtpm_basic_operations.py

```diff
@@ -61,7 +61,7 @@
 rm -rf ${TMPDIR}
 """
 
-@pytest.mark.small_vm
+@pytest.mark.debian_uefi_vm
```

Comment:
`jobs.py` says it is also supposed to work with a CentOS VM, and the requirements above also say a RPM distro can work as long as the `tpm2-tools` package is available (which we can see supported in `unix_vm_with_tpm2_tools`

Reply:
I've dropped the vtpm commit.